src/eventbridge/send.ts

```diff
@@ -32,17 +33,16 @@ const sendEvents = async (testResults: TestActivity[]): Promise<SendResponse> =>
 
     try {
       logger.debug(`event about to be sent: ${JSON.stringify(params)}`);
-      if (testResults[i].testTypeEndTimestamp !== '') {
-        // eslint-disable-next-line no-await-in-loop
-        const result = await eventbridge.putEvents(params).promise();
-        logger.info(
-          `${result.Entries.length} ${result.Entries.length === 1 ? 'event' : 'events'} sent to eventbridge.`,
-        );
+      // eslint-disable-next-line no-await-in-loop
+      const putResponse = await eventbridge.putEvents(params).promise();
+      if (putResponse.Entries[0].EventId) {
+        logger.info(`Result sent to eventbridge (testResultId: '${result.testResultId}', vin: '${result.vin}')`);
         sendResponse.SuccessCount++;
       } else {
-        logger.info(`Event not sent as test is not completed { ID: ${testResults[i].testResultId} }`);
+        throw new Error(putResponse.Entries[0].ErrorMessage);
```

Comment:
Would we really want to be throwing here, especially if we have multiple test results being processed

Reply:
Ahhhh